etc/helm/pachyderm/Chart.yaml

```diff
@@ -62,5 +62,6 @@ dependencies:
     condition: postgresql.enabled
   - name: loki-stack
     version: "2.8.1"
-    repository: "https://grafana.github.io/helm-charts"
+    repository: "{{ .Values.global.image.grafanaRegistry }}helm-charts"
```

Comment:
I don't think you can do this.  I think the value of this is only read during `helm dependency update`, but it can't handle variable expansion.

Reply:
reverted back.